configs/balrogworker

```diff
@@ -0,0 +1,25 @@
+{
+    "us-east-1": {
+        "type": "balrog-worker",
+        "domain": "srv.releng.use1.mozilla.com",
+        "ami": "ami-18abe670",
+        "subnet_ids": ["subnet-33a98358"],
+        "security_group_ids": ["sg-cf1027ab"],
+        "instance_type": "t2.micro",
```

Comment:
We may need to revisit the instance type - some of them may not be available in VPC. Nothing to change for now.